frontend/src/locales/scripts/en.json5

```diff
@@ -1051,6 +1051,37 @@
       creator: "Creator",
       source: "Source",
     },
+    pageTitle: {
+      tag: {
+        /**
+         * {tag} will be a dynamic value such as "cat". We cannot change its case or form.
+         * You can change the sentence to add more context and make the sentence
+         * grammatically correct, for instance, to "Audio tracks with the tag {tag}".
+         */
+        audio: "{tag} audio tracks | Openverse",
+
+        /**
+         * {tag} will be a dynamic value such as "cat". We cannot change its case or form.
+         * You can change the sentence to add more context and make the sentence
+         * grammatically correct, for instance, to "Images with the tag {tag}".
+         */
+        image: "{tag} images | Openverse",
+      },
+      source: {
+        /**
+         * {source} will be a dynamic value such as "Wikimedia". We cannot change its case or form.
+         * You can change the sentence to add more context and make the sentence
+         * grammatically correct, for instance, to "Audio tracks from {source}".
+         */
+        audio: "{source} audio tracks | Openverse",
+        /**
+         * {source} will be a dynamic value such as "Wikimedia". We cannot change its case or form.
+         * You can change the sentence to add more context and make the sentence
+         * grammatically correct, for instance, to "Images from {source}".
+         */
+        image: "{source} images | Openverse",
```

Comment:
Might be worth mentioning in these comments that they are used as browser page titles in particular. That might help clarify a preference for a succinct string rather than something fully descriptive or strictly gramatically correct, rather, it needs to be SEO friendly (I imagine length is a factor here). We can probably leave out the "You can change the sentence...", so that it's clear it does not need to be a complete sentence, but rather clarifying that it must be an SEO friendly phrase?

We normally interpolate `Openverse` as a variable. Any reason not to do that here?

Also, in Glotpress the curly praces are replaced with `###`, so it might be better to just quote `"tag"` (etc.) than reference it with the curly braces, which only make sense in the context of the original string.

Just suggestions below, nothing blocking here other than the Openverse intepolation if we should keep it.

Also: will the POT transformer get confused about the `|` in the string and think it's a plural form string, by any chance? Maybe that also needs to be interpolated or included in `{openverse}` as `" | Openverse` with the leading space? It could be better to have the strings here only be the titles, leaving out the `| Openverse` part altogether, and then interpolating the translated result into a plain template string, like `` `${i18n.t("pageTitle.tag.image", { tag })} | Openverse` ``? That way there is no chance the POT transformer gets confused about whether these are plural form strings and translators don't need to worry about or focus on spacing of the pipe and Openverse. If we need to use a template for the whole thing for e.g., RTL, then we can use two separate templates and interpolate them into each other, with a generic template for the page-title form: `"{pageTitle} {pipe} {openverse}"` for example. And then a note to the translator explaining what each variable will be and to reorder it as needed with Openverse always separated from the actual title by the pipe.

```suggestion
        /**
         * "tag" will be a dynamic value such as "cat". We cannot change its case or form.
         * Used as browser page title and must be SEO friendly
         */
        audio: "{tag} audio tracks | {openverse}",

        /**
         * "tag" will be a dynamic value such as "cat". We cannot change its case or form.
         * Used as browser page title and must be SEO friendly
         */
        image: "{tag} images | {openverse}",
      },
      source: {
        /**
         * "source" will be a dynamic value such as "Wikimedia Commons". We cannot change its case or form.
         * Used as browser page title and must be SEO friendly
         */
        audio: "{source} audio tracks | {openverse}",
        /**
         * "source" will be a dynamic value such as "Wikimedia Commons". We cannot change its case or form.
         * Used as browser page title and must be SEO friendly
         */
        image: "{source} images | {openverse}",
```